scripts/zones/AlTaieu/mobs/Absolute_Virtue.lua

```diff
@@ -4,20 +4,274 @@
+                if not entity.isLocked(jsa) then
+                    mob:setLocalVar(string.format('sp_%u', jsa), os.time())
+                    debug(string.format('%s using %d', mob:getName(), jsa))
+                    mob:useMobAbility(jsa)
+                end
+            end
+        elseif #xi.av.sps ~= 0 then
+            local sp = xi.av.sps[math.random(1, #xi.av.sps)]
+            mob:setLocalVar(string.format('sp_%u', sp), os.time())
+            debug(string.format('%s using %d', mob:getName(), sp))
+            mob:useMobAbility(sp)
+        end
+
+        -- TODO: minimum should be max of 2hr combos length
+        xi.av.nextsp = now + math.random(45, 90)
+    end
+end
+
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ABILITY_RESPONSE, 1)
+
+    --[[
+    mob:setMobMod(xi.mobMod.MAGIC_COOL, 20)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180)
+    mob:setMod(xi.mod.SOULEATERRES, 4)
+    mob:setMod(xi.mod.UFASTCAST, 100)
+    mob:setMod(xi.mod.MAIN_DMG_RATING, 150)
+    mob:setMod(xi.mod.ATT, 300)
+    mob:setMod(xi.mod.DEF, 300)
+    mob:setMod(xi.mod.MATT, 132)
+    mob:setMod(xi.mod.MDEF, 200)
+    mob:setMod(xi.mod.REFRESH, 500)
+    mob:setMod(xi.mod.REGAIN, 200)
+
+    mob:addImmunity(xi.immunity.SLEEP)
+    mob:addImmunity(xi.immunity.GRAVITY)
+    mob:addImmunity(xi.immunity.BIND)
+    mob:addImmunity(xi.immunity.STUN)
+    mob:addImmunity(xi.immunity.SILENCE)
+    mob:addImmunity(xi.immunity.PARALYZE)
+    mob:addImmunity(xi.immunity.BLIND)
+    mob:addImmunity(xi.immunity.SLOW)
+    mob:addImmunity(xi.immunity.POISON)
+    mob:addImmunity(xi.immunity.ELEGY)
+    mob:addImmunity(xi.immunity.REQUIEM)
+
+    mob:speed(60)
+    ]]--
+end
+
 entity.onMobSpawn = function(mob)
-    -- setMod
-    mob:setMod(xi.mod.REGEN, 500)
+    -- reset av
+    mob:setAnimationSub(1)
+
+    xi.av.regen = 250
+    xi.av.bracelets = false
+    xi.av.locks = {}
+    xi.av.sps =
+    {
+        xi.jsa.MIGHTY_STRIKES,
+        xi.jsa.BENEDICTION,
+        xi.jsa.HUNDRED_FISTS,
+        xi.jsa.MANAFONT,
+        xi.jsa.CHAINSPELL,
+        xi.jsa.PERFECT_DODGE, -- no combo
+        xi.jsa.INVINCIBLE,
+        xi.jsa.BLOOD_WEAPON, -- no combo
+        xi.jsa.SOUL_VOICE,
+        xi.jsa.MEIKYO_SHISUI,
+        xi.jsa.MIJIN_GAKURE,
+        xi.jsa.EES_AERN,
+        xi.jsa.CALL_WYVERN,
+        xi.jsa.FAMILIAR,
+        xi.jsa.ASTRAL_FLOW,
+    }
+
+    xi.av.braceletsps =
+    {
+        xi.jsa.CHAINSPELL,
+        xi.jsa.MIGHTY_STRIKES,
+        xi.jsa.MEIKYO_SHISUI,
+        xi.jsa.INVINCIBLE,
+        xi.jsa.CALL_WYVERN,
+    }
 
-    local jailerOfLove = GetMobByID(ID.mob.JAILER_OF_LOVE)
     -- Special check for regen modification by JoL pets killed
-    if jailerOfLove:getLocalVar('JoL_Qn_xzomit_Killed') == 9 then
-        mob:addMod(xi.mod.REGEN, -130)
+    local jol = GetMobByID(ID.mob.JAILER_OF_LOVE)
+    if jol:getLocalVar('JoL_Qn_xzomit_Killed') == 9 then
+        mob:addMod(xi.mod.REGEN, 125)
```

Comment:
This is adding regen to AV when JoLs pets are killed? I'm quite ignorant to AV but just making sure this is intended

Reply:
supposed to be reducing it, lost its signage. was `-130` got flipped to +125